episodes/12-homework.md

```diff
@@ -35,6 +35,7 @@ To prepare for our next session, please:
   Data Carpentry, or Library Carpentry lesson and
   read through it carefully.
   In the next two parts, you will use this to practice live coding/participatory instruction for 3 minutes.
+  These future exercises to not have built in prep time like the short 90second teaching practice already covered did, your preparation **before** we start tomorrow is all of your preparation time. 
```

Comment:
Two changes and one suggestion. 
1. "exercises to not have" -> "exercises do not have"
2. "90second" -> "90 second"

Suggest breaking into two sentences: "These future exercises do not have built in prep time like the short 90 second teaching practice we did earlier. Your preparation **before** we start tomorrow is all of your preparation time."